lib/upstream/zookeeper.js

```diff
@@ -14,10 +14,9 @@ class Zookeeper extends EventEmitter {
     super();
     this._conf = conf;
 
-    var zkPort = this._conf.zk.port;
-    var zkHost = this._conf.zk.host;
+    var {host, port} = this._conf.zk;
 
-    this._client = zookeeper.createClient(`${zkHost}:${zkPort}`);
+    this._client = zookeeper.createClient(`${host}:${port}`);
```

Comment:
And if you want to go crazy, we can do it [here](https://github.com/socrata-platform/geo-import/pull/2/files#diff-44fde2a35b73da13a9e6cd862bdda96dR41) too.